src/nurse/tests/management/commands/test_notifications.py

```diff
@@ -4,15 +4,27 @@
 from django.test import override_settings
 
 from nurse.management.commands import _notifications
+from nurse.models import Prescription, UserOneSignalProfile
 
 ONESIGNAL_APP_ID = "test_app_id"
 ONESIGNAL_API_KEY = "test_api_key"
 client_path = "nurse.management.commands._notifications.Client"
 
 
 class TestNotifications:
+    @pytest.mark.django_db
     def test_notify(self):
-        with mock.patch(
+        # Mock data
+        mock_prescriptions = [mock.Mock(), mock.Mock()]
+        mock_subscription_ids = ["123", "456"]
+
+        with mock.patch.object(
+            Prescription.objects, "expiring_soon", return_value=mock_prescriptions
+        ), mock.patch.object(
+            UserOneSignalProfile.objects,
+            "subscription_ids_for_users",
+            return_value=mock_subscription_ids,
```

Comment:
Load DB fixtures and test the DB side for real.
Only the `send_notification` call should be mocked to verify it was called properly

Reply:
done